src/Dynatrace.OpenTelemetry.Exporter.Metrics/Dynatrace.OpenTelemetry.Exporter.Metrics.csproj

```diff
@@ -19,6 +19,7 @@
   <ItemGroup>
```

Comment:
Please also bump the exporter version in line 8 above for these changes here.

Reply:
Fixed in [ce39559](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/24/commits/ce39559b55bd2fe42281178452f28bcb75cad113)! Bumped the patch version as no new features were added and public API stays the same.